partner_delivery_zone/readme/CONFIGURATION.rst

```diff
@@ -1,3 +1,4 @@
 To configure this module you need to:
 
-#. Go to *Sales > Configuration> Delivery Zones* and create any zones
+#. Go to Sales > Configuration > Delivery Zone and create required codes
```

Comment:
I think should go in Contacts > Configuration (and use plural in the menu name).

Reply:
This is not modified

Reply:
But we want the other way around.

Reply:
Readme CONFIGURATION file changed as per your request.

Reply:
But what about moving the menu to the Contacts app?

Reply:
ok I will do it